project/settings_pytest.py

```diff
@@ -20,6 +20,7 @@
 # Disable a bunch of third-party integrations by default.
 GEOCODING_SEARCH_URL = ""
 AIRTABLE_API_KEY = ''
+AIRTABLE_URL = ''
```

Comment:
Forgot to do this in #1341 and my test setup crashed because `settings_pytest.py` override my `AIRTABLE_API_KEY` but not my `AIRTABLE_URL`.